paywall/next.config.js

```diff
@@ -6,19 +8,32 @@ const withCSS = require('@zeit/next-css')
 
 const copyFile = promisify(fs.copyFile)
 
+const requiredConfigVariables = {
+  unlockEnv: process.env.UNLOCK_ENV || 'dev',
+  httpProvider: process.env.HTTP_PROVIDER,
+  readOnlyProvider: process.env.READ_ONLY_PROVIDER,
+  paywallUrl: process.env.PAYWALL_URL,
+  paywallScriptPath: process.env.PAYWALL_SCRIPT_PATH,
+  locksmithHost: process.env.LOCKSMITH_URI,
+  unlockAddress: process.env.UNLOCK_ADDRESS,
+}
+
+Object.keys(requiredConfigVariables).forEach(configVariableName => {
+  if (!requiredConfigVariables[configVariableName]) {
+    if (['dev', 'test'].indexOf(requiredConfigVariables.unlockEnv) > -1) {
```

Comment:
This will add a lot of logging messages in our logs... which means we need to make sure these vars are set!